### DISCUSSION TITLE
When is the next release date?

### DISCUSSION THREAD

**Discussion by ashwinvinjimur** [py-pdf/fpdf2#1319] [General]
Hi, 

Thank you so much for all the good work! I'm looking to see when the next expected release date is? 

I'm specifically interested in using the `allow_extra_pages` feature implemented as part of #1188.

**Comment by Lucas-C** [comment]
Hi @ashwinvinjimur

Thank you for your kind words 🙂

I think we could plan a release before the end of the year.

Maybe we could include 1 or 2 PR currently in progress:
* maybe this bugfix: https://github.com/py-pdf/fpdf2/pull/1316?
* depending on @andersonhc, maybe https://github.com/py-pdf/fpdf2/pull/1305?
* maybe even this one: https://github.com/py-pdf/fpdf2/pull/1300?

I think we can aim for the week of the 16th of December for a release.

Wish you a happy holiday season 🎄☃️ 𐂂

**Comment by ashwinvinjimur** [comment]
Thank you @Lucas-C and wish you a very happy holiday season as well! :)

**Reply by Lucas-C** [reply]
Hi @ashwinvinjimur

A new release of `fpdf2` has been published today: https://github.com/py-pdf/fpdf2/releases

**Reply by ashwinvinjimur** [reply]
Hi @Lucas-C - Thank you so much! Upgrading now :)
